### PR TITLE
description: Add structure for Lectures teaching stage

### PR DESCRIPTION
**Pull request by razvand**
Create structure and key ideas for Lectures teaching stage. Each item features bullet points to be further developed after feedback.

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -8,6 +8,70 @@ Each topic in this file contains a small description and links to the files cove
...
+- rough high-level structure to be adapted to each lecturer's personality and view of things
+- conclusion
+- layered approach: top-to-bottom
+
+#### Connection to Other Stages
+
+- lecture before lab, seminar
+- demos during both lectures and labs
+- direct references between lectures and labs (forward reference from lectures to labs)
+- direct references between assignments and lectures
```
Comment (teodutu) [inline description.md:60]: Add that assignments should build on top of lectures and labs.

File: description.md [hunk 2]
```diff
@@ -8,6 +8,70 @@ Each topic in this file contains a small description and links to the files cove
...
+#### Best Practices
+
+- layered approach
+- real-world use cases
+- personal lecturer experiences & anecdotes
+- lecturer to be part of projects & activities related to the topic
+- materials to be OER
+- encourage contributions
+- quiz at the beginning based on earlier course
+- some feedback form (not to often, because it will bore participants)
```
Comment (teodutu) [inline description.md:71]:
```suggestion
- some feedback form (not too often, because it will bore participants)
```

File: description.md [hunk 3]
```diff
@@ -8,6 +8,70 @@ Each topic in this file contains a small description and links to the files cove
...
+- record sessions, apply feedback, reiterate content
+- have demos
```
Comment (teodutu) [inline description.md:73]:
```suggestion
- have demos
- use the blackboard, draw figures on the spot
- avoid slides with images, definitely avoid slides with text
```

**Review by teodutu — CHANGES_REQUESTED** [review]
I gave some thoughts on what you've written so far. See if you agree. I'll leave more reviews as you add more content.

File: description.md [hunk 4]
```diff
@@ -8,6 +8,70 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.
```
[the remark this answers is not in the document]
Comment (razvand) [reply description.md:11]: Elaborate on the last sentence being wrong?

File: description.md [hunk 5]
```diff
@@ -8,6 +8,70 @@ Each topic in this file contains a small description and links to the files cove
...
+- getting participants to be (inter)active
+- providing high-quality material before, during and after lecture
+- correlating with other stages for concerted knowledge transfer
+- exciting participants about the topic
+- building a community of participants around the topic
+
+#### Lecture Metrics
+
+- attendance
+- interaction
```
[the remark this answers is not in the document]
Comment (razvand) [reply description.md]: You can measure interaction why the amount of questions you get, students who take part in it.

File: description.md (hunk 1 above)
Comment (razvand) [reply description.md]: for sure

File: description.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (teodutu) [reply description.md:11]: > is limited to on conversation only

What is "on conversation"? It doesn't sound right. Reading it again, it seems it should be without "on": "is limited to conversation only"

File: description.md [hunk 6]
```diff
@@ -3,11 +3,44 @@
...
 
-
 ## Teaching Stages
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.
+Labs, for example, allow each participant to focus on one item they could work on.
+Seminars are close to lectures, except that they typically involve a smaller group of people and are naturally interactive, with the relationship being less asymmetrical.
```
Comment (Alex-deVis) [inline description.md:13]: I think this build-up does not lead anywhere, consider talking about labs and seminars to introduce the need for materials (in multiple forms) to make lectures as effective.

File: description.md [hunk 7]
```diff
@@ -3,11 +3,44 @@
...
+
+Materials are required for delivering lectures.
+In the following sections, we detail steps in preparing and (re)using lecture materials in Open Education Hub.
+
+We organize lecture materials according lecture-specific activities:
+
+* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.
+  They consist of:
+  * **Slides** are the typical structured and simplified format for organizing ideas.
+    The lecturer typically walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
```
Comment (Alex-deVis) [inline description.md:23]:
```suggestion
    The lecturer walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
```
`typical` is repeated.

File: description.md (hunk 6 above)
Comment (RazvanN7) [reply description.md:13]: On a second thought, you could just rephrase this to: "Lectures are different from practical sessions (link to practical sessions) by ...".

File: description.md [hunk 8]
```diff
@@ -3,11 +3,44 @@
...
+  * **Media files** are generally images or videos that represent a concept in an easier to grasp format.
```
Comment (RazvanN7) [inline description.md:24]: I don't think this category is necessary. You can say that media and reading are embedded into slides.

**Review by RazvanN7 — CHANGES_REQUESTED** [review]
Looks good, but need to comment a bit on how the materials for lectures are organized in the repo.

**Comment by razvand** [comment]
> Looks good, but need to comment a bit on how the materials for lectures are organized in the repo.

I don't think so. That is mentioned in the `Types of Content` section, isn't it? Or you mean how one would store the selected items into given lecture sessions?

File: description.md (hunk 6 above)
Comment (razvand) [reply description.md]: @Alex-deVis , @RazvanN7 , indeed. I rephrased this.

File: description.md (hunk 8 above)
Comment (razvand) [reply description.md]: Maybe some media may not be embedded. I think I can leave it separate, but mention that media is typically embedded into slides.

File: description.md [hunk 9]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interactivity is important but is limited by the larger audience: not all participants will be able to actively engage in conversations;
+however, items that don't require conversation, such as quizzes, are a good way to engage all participants, as we discuss below.
```
Comment (Alex-deVis) [inline description.md:37]:
```suggestion
however, items that don't require conversation, such as [quizzes](#quizzes), are a good way to engage all participants, as we discuss below.
```

File: description.md [hunk 10]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+Materials are required for delivering lectures.
+In the following sections, we detail steps in preparing and (re)using lecture materials in Open Education Hub.
+
+#### Lecture Materials
+
+We organize lecture materials according to lecture-specific activities.
+We reference them as presented in the ["Types of Content" section](#types-of-content).
+
+* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.
+  They consist of:
```
Comment (Alex-deVis) [inline description.md:49]:
```suggestion

  They consist of:
```
Without an empty line before the indented line, there is no change in the view. Is this for stylistic reasons only?

File: description.md [hunk 11]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+
+* **Assessment materials** are used to evaluate knowledge and skills of participants, and to provide feedback on spots they need to focus more on.
+  Typically they are to be used live during lectures, with their results providing immediate feedback and triggering discussions.
+  They consist of [**Quizzes**](#quizzes).
+
+* **Offline self-study materials** are to be used by participants outside lectures.
+  They are detailed in the ["Self-Study" section](#self-study).
+
+As other types of content, lecture materials are publicly available, and also open to contributions.
+Lecture materials are meant to be used live by teachers during lectures and then offline by learners, to improve gained knowledge and skills.
```
Comment (Alex-deVis) [inline description.md:72]:
```suggestion
Lecture materials are meant to be used live by educators during lectures and then offline by learners, to improve gained knowledge and skills.
```

File: description.md [hunk 12]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+The dedicated teacher will be careful to adjust the types of content to the topic being presented and to the audience.
+
+#### Evaluating Lecture Content
+
+It is difficult to evaluate lecture content without including the teacher delivery method itself.
+A charismatic teacher will likely get good student feedback even with low-quality lecture content.
+
+Even so, there are several metrics that we recommend to be considered:
+
+* Good quiz grades (during lectures) are generally a sign that the respective topics are well presented.
```
Comment (Alex-deVis) [inline description.md:97]:
```suggestion
- Good quiz grades (during lectures) are generally a sign that the respective topics are well presented.
```
We should decide on a way to present lists; other PRs use **-**.

File: description.md [hunk 13]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+#### Lecture Materials
+
+We organize lecture materials according to lecture-specific activities.
+We reference them as presented in the ["Types of Content" section](#types-of-content).
+
+* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.
+  They consist of:
+
+  * [**Slides**](#slides) are the typical structured and simplified format for organizing ideas.
+    The lecturer walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
```
Comment (teodutu) [inline description.md:52]:
```suggestion
    The lecturer may walk through slides and present information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
```

File: description.md [hunk 14]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+    Diagrams, photos, recordings of actions / processes, recorded simulations fall under this category.
+    They can be used either stand-alone or embedded into slides.
+
+* **Practical support materials** are used to demonstrate actual real-world applications of presented concepts.
+  The lecturer is the one who uses the practical support materials to do live demonstrations and highlight concrete use cases.
+  This may not be possible for certain topics, either because they are highly theoretical or because it is difficult to have a live setup.
+  Where a live demonstration is not possible, a video recording (as a visual support material) may be used.
+  They consist of [**Demos**](#demos).
+
+* **Assessment materials** are used to evaluate knowledge and skills of participants, and to provide feedback on spots they need to focus more on.
```
Comment (teodutu) [inline description.md:64]:
```suggestion
* **Assessment materials** are used to evaluate the knowledge and skills of participants, and to provide feedback on spots they need to focus more on.
```

File: description.md [hunk 15]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+
+Even so, it is recommended that the types of materials shown above be intertwined during an actual lecture.
+Slides are good for presenting general topics, but tend to become tern and difficult to follow.
+As such, spreading demos and quizzes between slides is great to engage the audience and also provide a practical overview of items and an insight into topics that deserve continued attention.
+
+One possibility is to have lectures focused primarily on demos and on quizzes and slides to be used sparringly, only as topic highlights.
+
+In the ideal case, each topic in a lecture will be a balanced mix of demos, quizzes, slides and media file.
+The dedicated teacher will be careful to adjust the types of content to the topic being presented and to the audience.
+
```
Comment (teodutu) [inline description.md:89]: You usually ditch slides altogether and only use demos and quizzes. You should present this approach as well. Maybe split them into subsections? Although the heading would be 5 `#`s, so it would probably be too much.

Either way, I would emphasize experimenting with various mixes according to the type of audience. Maybe people from different cultures react differently to "stimuli" such as quizzes or demos.

File: description.md [hunk 16]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+#### Evaluating Lecture Content
+
+It is difficult to evaluate lecture content without including the teacher delivery method itself.
+A charismatic teacher will likely get good student feedback even with low-quality lecture content.
+
+Even so, there are several metrics that we recommend to be considered:
+
+* Good quiz grades (during lectures) are generally a sign that the respective topics are well presented.
+
+* Good assignment grades and fewer technical assignment clarfications mean that the topics for assignments were well covered during lectures.
```
Comment (teodutu) [inline description.md:99]:
```suggestion
* Good assignment grades and fewer technical assignment clarifications requested by students mean that the topics for assignments were well covered during lectures.
```

File: description.md [hunk 17]
```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
...
+
+* Website accesses for particulare materials signal the topic is deemed well presented by students (and others - as it is public content).
```
Comment (teodutu) [inline description.md:101]:
```suggestion
* Website accesses for particular materials signal the topic is deemed well presented by students (and others - as it is public content).
```
Or rather that it's of interest to them, possibly even badly presented. If something goes smoothly and is clear, why would you revisit it?

**Review by teodutu — CHANGES_REQUESTED** [review]
I think you preached a certain method of teaching (slides + quizzes + demos) a bit too mechanically, with little room for variation. You might want to relax the wording here and there.

File: description.md (hunk 10 above)
Comment (razvand) [reply description.md:49]: @Alex-deVis , I don't understand this.

File: description.md (hunk 15 above)
Comment (razvand) [reply description.md:89]: Agree. Will augment that.
